db/compaction/compaction_picker.cc

```diff
@@ -39,7 +39,14 @@ bool FindIntraL0Compaction(const std::vector<FileMetaData*>& level_files,
                            size_t min_files_to_compact,
                            uint64_t max_compact_bytes_per_del_file,
                            uint64_t max_compaction_bytes,
-                           CompactionInputFiles* comp_inputs) {
+                           CompactionInputFiles* comp_inputs,
+                           SequenceNumber oldest_mem_seqno) {
```

Comment:
Another naming nit: `smallest_unflushed_seqno` would make it clearer that it is the fact that this seqno is unflushed which is important.